### ISSUE TITLE
New Term: Inability to protrude lips

### ISSUE THREAD

**Issue by CorneliusDreyling** [obophenotype/human-phenotype-ontology#10570]
**Preferred term label:**
Inability to protrude lips

**Synonyms**
none

**Definition (free text, please give PubMed ID)**
Due to facial muscle weakness inability to protrude lips. (https://doi.org/10.1007/s00415-020-10281-z)

**Parent term (use hpo.jax.org/app)**
Weakness of facial musculature HP:0030319

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA:269

**Your nano-attribution (ORCID)**
0009-0004-9942-3886

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6001010
- label: Inability to protrude lips
- parents: [Weakness of facial musculature (HP:0030319)](https://hpo.jax.org/app/browse/term/HP:0030319)
- pmid: PMID:30804066
- def: Lack of ability to purse or pucker the lips.
- comment: The anomaly can be related to abnormalities of the muscles, nerves, or structure of the face.
- orcid: orcid:0009-0004-9942-3886
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10570
- synonyms:
